packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/categories/LinkToCategoriesTests.java

```diff
@@ -327,6 +328,106 @@ public void testLinkContentToCategory_usingRootCategoryAndExpect400()
         restClient.assertStatusCodeIs(BAD_REQUEST);
     }
 
+    /**
+     * Try to link and unlink content from a created category
+     */
+    @Test(groups = {TestGroup.REST_API})
+    public void testUnlinkContentFromCategory()
+    {
+        STEP("Link content to created category and expect 201");
+        final RestCategoryLinkBodyModel categoryLink = createCategoryLinkWithId(category.getId());
+        final RestCategoryModel linkedCategory = restClient.authenticateUser(user).withCoreAPI().usingNode(file).linkToCategory(categoryLink);
+
+        restClient.assertStatusCodeIs(CREATED);
+        linkedCategory.assertThat().isEqualTo(category);
+
+        STEP("Verify that category is present in file metadata");
+        RestNodeModel fileNode = restClient.authenticateUser(user).withCoreAPI().usingNode(file).getNode();
+
+        fileNode.assertThat().field(ASPECTS_FIELD).contains("cm:generalclassifiable");
+        fileNode.assertThat().field(PROPERTIES_FIELD).contains("cm:categories");
+        fileNode.assertThat().field(PROPERTIES_FIELD).contains(category.getId());
+
+        STEP("Unlink content from created category and expect 204");
+        restClient.authenticateUser(dataContent.getAdminUser()).withCoreAPI().usingNode(file).unlinkFromCategory(category.getId());
+        restClient.assertStatusCodeIs(NO_CONTENT);
+
+        STEP("Verify that category isn't present in file metadata");
+        fileNode = restClient.authenticateUser(user).withCoreAPI().usingNode(file).getNode();
+
+        fileNode.assertThat().field(ASPECTS_FIELD).notContains("cm:generalclassifiable");
+        fileNode.assertThat().field(PROPERTIES_FIELD).notContains("cm:categories");
+        fileNode.assertThat().field(PROPERTIES_FIELD).notContains(category.getId());
+    }
+
+    /**
+     * Try to link content to multiple categories and try to unlink content from a single category
+     * Other categories should remain intact and file should keep having "cm:generalclassifiable" aspect
+     */
+    @Test(groups = {TestGroup.REST_API})
+    public void testUnlinkContentFromCategory_multipleLinkedCategories()
+    {
+        STEP("Create second category under root");
+        final RestCategoryModel secondCategory = prepareCategoryUnderRoot();
+
+        STEP("Link content to created categories and expect 201");
+        final List<RestCategoryLinkBodyModel> categoryLinks = List.of(
+                createCategoryLinkWithId(category.getId()),
+                createCategoryLinkWithId(secondCategory.getId())
+        );
+        final RestCategoryModelsCollection linkedCategories = restClient.authenticateUser(user).withCoreAPI().usingNode(file).linkToCategories(categoryLinks);
+
+        restClient.assertStatusCodeIs(CREATED);
+        linkedCategories.getEntries().get(0).onModel().assertThat().isEqualTo(category);
+        linkedCategories.getEntries().get(1).onModel().assertThat().isEqualTo(secondCategory);
```

Comment:
These assertions are duplicating existing tests above. If you want to make an assertion about the status code I think that's ok but it's probably unnecessary to do more than that.